GOVERNANCE.md

```diff
@@ -47,3 +47,41 @@ To honor the past owners, here's the history of the owners and the time they ser
 
 - 2019-07-25 ~ 2021-09-28
   - [Laszlo Fogas](https://github.com/laszlocph)
+
+## Access
+
+The project has multiple accounts and systems. Owners have admin access to all systems. Maintainers may be granted additional access.
+
+### Accounts
+
+- [GitHub Org](https://github.com/woodpecker-ci/)
+- [Codeberg Org](https://codeberg.org/woodpecker-ci)
+- [Codeberg Plugins Org](https://codeberg.org/woodpecker-plugins)
+- [OpenCollective](https://opencollective.com/woodpecker-ci)
+- [Docker](https://hub.docker.com/u/woodpeckerci)
```

Comment:
```suggestion
- [Docker](https://hub.docker.com/u/woodpeckerci)
- [QuayIO](https://quay.io/organization/woodpeckerci)
```

Reply:
<img width="1601" height="1440" alt="image" src="https://github.com/user-attachments/assets/861a9577-f3de-49d1-bfad-1db1c9b39033" />
<img width="1196" height="668" alt="image" src="https://github.com/user-attachments/assets/3d4f5a7f-f52b-411d-b631-ce4412ceafca" />

Reply:
NOTE: the quai repo is as backup to not be threatened by `docker inc.` if they at some point want to do shit again